MobileCenterCrashes/MobileCenterCrashesTests/MSCrashesTests.mm

```diff
@@ -181,23 +178,24 @@ - (void)testCreateBufferFile {
   // When
   NSString *testName = @"afilename";
   [self.sut createBufferFileWithName:testName forPriority:MSPriorityHigh];
-  
+
   // Then
-  NSString *priorityDirectory = [self.sut.logBufferDir stringByAppendingFormat:@"/%ld/", MSPriorityHigh];
 
+  // Wait for a little bit to make sure the files have been created.
+  [NSThread sleepForTimeInterval:0.1];
```

Comment:
Is this reliable? If you add a synchronous method to create files and call it asynchronously, you can use synchronous method for test and remove this line I guess.